tag/nitag.yml

```diff
@@ -0,0 +1,1816 @@
+      keywords:
+        type: array
+        items:
+          type: string
+      collectAggregates:
+        type: boolean
+    example:
+      type: DOUBLE
+      properties:
+        key1: value1
+        key2: value2
+      path: foo
+      keywords: [fooKeyword, barKeyword]
+      collectAggregates: true
+  TagUpdate:
+    description: Object containing a tag's path, and one or more updates to be applied to it
+    type: object
+    title: Tag Update
+    properties:
+      path:
+        type: string
+      updates:
+        type: array
+        items:
+          $ref: '$/definitions/TimestampedTagValue'
+    example:
+      path: foo
+      updates:
+        - value:
+            type: DOUBLE
+            value: "3.14"
+          timestamp: '2018-09-04T18:45:08Z'
+        - value:
+            type: DOUBLE
+            value: "5.0"
+          timestamp: '2018-09-04T18:45:08Z'
+  SubscriptionUpdate:
+    description: Updates for a subscriptions
+    type: object
+    title: Subscription Update
+    properties:
+      updates:
+        type: array
+        items:
+          $ref: '#/definitions/SubscriptionTagUpdate'
+      subscriptionId:
+        type: string
+      nonexistentTags:
+        description: List of tags which did not exist
+        type: array
+        items:
+          type: string
+      deletedTags:
+        description: List of tags which have been deleted
+        type: array
+        items:
+          type: string
+      createdTags:
+        description: List of tags which have been created
+        type: array
+        items:
+          type: string
+      alias:
+        description: Name of the subscription
+        type: string
+  SubscriptionTagUpdate:
+    description: Update details for a tag subscription
+    type: object
+    title: Subscription Tag Update
+    properties:
+      value:
+        description: Updated tag value
+        type: string
+      type:
+        type: string
+        enum: *TYPES
+      timestamp:
+        description: Updated timestamp
+        type: string
+      tag:
+        $ref: '#/definitions/Tag'
+      aggregates:
+        $ref: '#/definitions/TagAggregates'
+    example:
+      value: "5.0"
+      type: DOUBLE
+      timestamp: '2018-09-04T18:45:08Z'
+      tag:
+        type: DOUBLE
+        properties:
+          key1: value1
+          key2: value2
+        path: foo
+        keywords: [fooKeyword, barKeyword]
+      aggregates:
+        sum: "10.5"
+        min: "0.0"
+        max: "5.0"
+        mean: "2.0"
+        count: 5
+  Subscription:
+    description: Object describing a subscription's metadata
+    type: object
+    title: Subscription
+    properties:
+      updatesOnly:
+        description: Whether to notify only about updates
+        type: boolean
+      tags:
+        description: List of subscribed tag paths
+        type: array
+        items:
+          type: string
+      alias:
+        description: Name of the subscription
+        type: string
```

Comment:
Should all three of these be required? Or maybe updatesOnly would be optional (in which case we should provide a default).

Reply:
Alias doesn't seem to be required, but updatesOnly defaults to false, and the paths are required.